<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptWindow</class>
 <widget class="QDialog" name="ScriptWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python Interpreter</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="buttonPane">
    <item>
     <widget class="QPushButton" name="openEditorButton">
      <property name="text">
       <string>Open Editor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runFileButton">
      <property name="text">
       <string>Run File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>550</y>
     <width>100</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Run Script</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="resetButton">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="interpreterLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="displayLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
